Duplicati/Server/webroot/ngax/scripts/controllers/SystemSettingsController.js

```diff
@@ -108,7 +108,7 @@ backupApp.controller('SystemSettingsController', function($rootScope, $scope, $l
                 if ($scope.updateChannel != $scope.originalUpdateChannel)
                     AppService.post('/updates/check');
 
-                $location.path('/');
+                $route.reload();
```

Comment:
I get a javascript error `ReferenceError: $route is not defined` in SystemSettingsController.js when running the code and trying to save the settings. It still saves the settings but nothing happens in the UI.

Reply:
I accidentally discarded the hunk with that addition when removing something else, will update